submarine-server/server-core/src/main/java/org/apache/submarine/server/job/JobServiceNotFoundJobException.java

```diff
@@ -17,12 +17,10 @@
  * under the License.
  */
 
-package org.apache.submarine.server.api.exception;
+package org.apache.submarine.server.job;
 
-public class UnsupportedJobTypeException extends Exception {
-  private static final long serialVersionUID = 4752254162145918312L;
-
-  public UnsupportedJobTypeException() {
-    super("Unsupported Job Type Exception");
+public class JobServiceNotFoundJobException extends JobServiceException {
+  public JobServiceNotFoundJobException() {
```

Comment:
Now the extended `*Exception` makes the system very complicated.
It is sufficient to use `SubmarineRuntimeException` in the commons-utils module.